java/ql/lib/semmle/code/java/security/SensitiveApi.qll

```diff
@@ -3,17 +3,45 @@
  */
 
 import java
+private import semmle.code.java.dataflow.DataFlow
+private import semmle.code.java.dataflow.ExternalFlow
+
+/**
+ * A node which represents the use of a credential.
```

Comment:
From [qldoc-style-guide.md#qldoc-for-classes](https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#qldoc-for-classes): 
> Use "that", not "which".

```suggestion
 * A node that represents the use of a credential.
```